components/do_verify_email/do_verify_email.tsx

```diff
@@ -55,7 +55,12 @@ export default class DoVerifyEmail extends React.PureComponent<Props, State> {
         if (this.props.isLoggedIn) {
             GlobalActions.redirectUserToDefaultTeam();
         } else {
-            browserHistory.push('/login?extra=verified&email=' + encodeURIComponent((new URLSearchParams(this.props.location.search)).get('email') || ''));
+            let link = '/login?extra=verified&email=' + encodeURIComponent((new URLSearchParams(this.props.location.search)).get('email') || '');
```

Comment:
Should we split this one too for some added clarity? (like with `redirectTo` below)